pkg/engine/variables/operator/in.go

```diff
@@ -0,0 +1,100 @@
+package operator
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/go-logr/logr"
+	"github.com/nirmata/kyverno/pkg/engine/context"
+)
+
+//NewInHandler returns handler to manage In operations
+func NewInHandler(log logr.Logger, ctx context.EvalInterface, subHandler VariableSubstitutionHandler) OperatorHandler {
+	return InHandler{
+		ctx:        ctx,
+		subHandler: subHandler,
+		log:        log,
+	}
+}
+
+//InHandler provides implementation to handle NotIn oerator
+type InHandler struct {
+	ctx        context.EvalInterface
+	subHandler VariableSubstitutionHandler
+	log        logr.Logger
+}
+
+//Evaluate evaluates expression with In Operator
+func (in InHandler) Evaluate(key, value interface{}) bool {
+	var err error
+	//TODO: decouple variables from evaluation
+	// substitute the variables
+	if key, err = in.subHandler(in.log, in.ctx, key); err != nil {
+		// Failed to resolve the variable
+		in.log.Error(err, "Failed to resolve variable", "variable", key)
+		return false
+	}
+	if value, err = in.subHandler(in.log, in.ctx, value); err != nil {
+		// Failed to resolve the variable
```

Comment:
remove obvious comments...this does not provide any additional information to the reader

Reply:
updated accordingly.